Add add-on url to rejection emails

kewisch opened the pull request:
When reading emails on amo-editors that relate to a rejected add-on I didn't personally review, I either have to guess the review link, google the add-on name, or for unlisted add-ons use mxr to find the add-on.

It would be much easier for me rejection/needinfo/approval emails sent to the developer would include the link to the add-on. From there I can easily get to the review page using the AMO Admin Assistant.

I'm open to other places for the url, I could put it into a new sentence under where it is now, e.g.
 `To view the status of your add-on, visit https://addons.mozilla.org/en-US/developers/addon/xxxx/`

r? @jvillalobos

jvillalobos commented:
r+ if `{{ addon_url }}` points to the developer page rather than the public page.

kewisch commented:
Oh right, it points to the public page except for with unlisted add-ons. I'll switch it to the dev url.

kewisch commented:
I've added a test now that makes sure the right urls are contained, partially to get acquainted with writing them. If you think this test will only get in the way I can remove it again.

I've also removed the `.` after the urls for consistency. For some templates there was a period, for others there was not. Now there is never one, to make sure email linkifiers don't add that period to the url.

jvillalobos commented:
Looks good to me, but someone more familiar with the repo should look at the code. Maybe @wagnerand?

wagnerand commented:
Nice test!

lgtm, r+
